testing/test-utils/src/lib/utils/logging.ts

```diff
@@ -0,0 +1,8 @@
+import type { Logger } from '@poppinss/cliui';
+
+export function getLogMessages(logger: Logger): string[] {
```

Comment:
I noticed that `logs[0]` is the main use case. There could be a new/adjusted helper function to get Nth log message. Then a string would be returned directly.

Reply:
you could add it to #552